wallet-new/test/MarshallingSpec.hs

```diff
@@ -3,11 +3,13 @@ module MarshallingSpec where
 import           Universum
 
 import           Cardano.Wallet.API.V1.Errors (WalletError)
+import           Cardano.Wallet.API.V1.Migration.Types (Migrate (..))
```

Comment:
The modules was restructured in order to be able to import it here as otherwise I was getting overlapping instances:
```
/home/akegalj/projects/serokell/cardano-sl/wallet-new/test/MarshallingSpec.hs:25:9: error:
    • Overlapping instances for ToJSON BackupPhrase
        arising from a use of 'aesonRoundtripProp'
      Matching instances:
        instance ToJSON BackupPhrase
          -- Defined in 'Cardano.Wallet.Orphans.Aeson'
        instance ToJSON BackupPhrase
```